jekyll/_cci2/contexts.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: classic-docs
+title: "CircleCI Contexts"
+short-title: "Using Contexts"
+description: "Secured, cross-project resources"
+categories: [configuring-jobs]
+order: 41
+---
+
+**NOTE:** As of July 2017 Contexts are launched in their MVP state. Some future features of Contexts are called out below as "Coming Soon". We encourage your feedback either on [our Discuss forum](https://discuss.circleci.com/t/contexts-feedback/13908) or in the Pull Request for this document.
+
+<hr>
+
+## Contexts Overview
+Contexts provide centrally managed, secured resources for use in jobs. They are primarily used for sharing environment variables across projects.
```

Comment:
Kinda buries the lede. whenever you talk about it internally you speak way more about the future of Context.  "Sharing env vars across projects" isn't really the primary use - it's the current v1 use.

here's some proposed text.


Contexts provide a new way to think about managing your build environments.

Contexts involve Resources (e.g., secrets/passwords/environment variables, fleets/machines) and Security Rules (which objects (e.g., projects/groups) can use which resources).  It is centrally managed at the organization level in the UI and gets invoked through config on a job-by-job basis

Contexts are designed to make it more convenient to add, update, and share environment variables within an organization.  As opposed to setting environment variables in the UI on a project-level, Contexts are set in UI at an organization level and are invoked by jobs through configuration files. 

This leads to new opportunities related  to security and permissions such that it will be easy to manage individual/group access to resources (secrets, fleets, machines, etc) wholly outside of your VCS-provider and such that organizations can prevent code (on a per-job basis) from running on certain resources.